Assessment.Test/Assessment.Test/ViewModels/LoginViewModel.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Assessment.Test.Models;
+using Assessment.Test.ViewModels.Base;
+using Assessment.Test.Views;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Text;
+using Xamarin.Forms;
+
+namespace Assessment.Test.ViewModels
+{
+    public class LoginViewModel : INotifyPropertyChanged
+    {
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        
+
+        public Command cmdLogin { get; set; }
+
+        public LoginViewModel()
+        {
+            cmdLogin = new Command(gotoHomePage);
+            userList.Add(new UserModel { userName = "user1", password = "12345" });
+            userList.Add(new UserModel { userName = "user2", password = "54321" });
+        }
+
+        private void gotoHomePage(object obj)
+        {
+            if (login(userName,password))
+            {
+                App.Current.MainPage.Navigation.PushAsync(new StockTransferView());
```

Comment:
Create page and bind the ViewModel and push to the navigation stack.Don't push directly page like this